jobs/async-upload/job/config.py

```
@@ -8,23 +8,33 @@
+    CreateModelIntent,
+    CreateVersionIntent,
+    UpdateArtifactIntent,
+    ConfigMapMetadata,
+    RegisteredModelMetadata,
+    ModelVersionMetadata,
+    ModelArtifactMetadata,
 )
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_METADATA_CONFIGMAP_PATH: Final[str] = "/etc/model-metadata"
```

Ruff (0.12.2)

Undefined name `Final`

(F821)